scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DenotationOps.scala

```diff
@@ -104,10 +105,13 @@ trait DenotationOps { self: DatabaseOps =>
         synthetic.text -> names.toList
       }
     }
+    private def overrides(includingOverride: Boolean): List[MSymbol] =
+      if (includingOverride) gsym.overrides.map(_.toSemantic)
+      else Nil
```

Comment:
How about we check `config.overrides` here directly? This will allow to get rid of the `includingOverride` parameters in `overrides` and `toDenotation`.